src/Components/Custom-Button/index.js

```diff
@@ -1,7 +1,28 @@
 import React from "react";
+import styled, { css } from "styled-components";
+import PropTypes from "prop-types";
 
-const CustomButton = ({ title, className }) => {
-  return <button className={className}>{title}</button>;
+const Button = styled.button`
+  color: ${props => (props.color ? props.color : "#000")};
+  font-size: 1rem;
+  margin: 0.5rem;
+  padding: 0.25rem 1rem;
+  border: 0.2rem solid ${props => (props.color ? props.color : "#000")};
+  border-radius: 0.2rem;
+`;
+
+const CustomButton = ({ title, className, color }) => {
```

Comment:
I hope I am not asking too much but it's cleaner if we use implicit return such as if a function only returns something:

```
// instead of 
const someComponent = (props) => {
  return (
    <div>I am a component the uses the return keyword</div>
  )
}

// we can simply:

const anotherComponent = (props) => (
 <div>I do not use the return keyword since I just return myself</div>
)
```

Reply:
No problem. It's a great suggestion because I didn't realize I could make it cleaner